cmd/shadowsocks-httpget/httpget.go

```diff
@@ -76,9 +81,94 @@ func get(connid int, uri, serverAddr string, rawAddr []byte, cipher *ss.Cipher,
 	}
 }
 
+func dialSocks5(targetAddr, proxy string) (conn net.Conn, err error) {
```

Comment:
this should be added into `shadowsocks` package

Reply:
Done.